master-firmware/src/udp_topic_injector.c

```diff
@@ -10,6 +10,9 @@ static void udp_topic_injector_thd(void *p)
 
     struct netconn *conn;
 
+    chThdSetPriority(HIGHPRIO);
```

Comment:
Can you add a comment explaining why HIGHPRIO is required there ?

Reply:
https://github.com/cvra/robot-software/blob/master/master-firmware/src/priorities.h ?

Reply:
it's confusing, why not create it as `HIGHPRIO` below? and it's good point, why`HIGHPRIO`? I'd expect debugging tasks to be `NORMALPRIO` or `LOWPRIO`

Reply:
I am not sure if it warrants being entered in the priority table. This implementation can only work if this thread is of the higher priority, otherwise messages will be lost. Plus priorities.h is not even used by a lot of people.

Reply:
> This implementation can only work if this thread is of the higher priority

Sad. We can't do `chThdSetPriority(HIGHESTPRIO());` :crying_cat_face:. Why do you need the higher priority? I'm worried about the control/odometry threads. :thinking: 

> Plus priorities.h is not even used by a lot of people.

Sure, but if it's the way to go, we can make every thread use it. RE-FAC-TO-RING!G

Reply:
HIGHPRIO is by defintion the highest priority supported by the ChibiOS kernel IIRC. We need the highest priority here to avoid dropping messagebus messages, although that might be acceptable. I don't think the odometry thread will be starved by this more than anything, the position manager is already running at NORMALPRIO, along with everything else.

> Sure, but if it's the way to go, we can make every thread use it. RE-FAC-TO-RING!

Sure, but then we should file a bug and someone should implement it.

Reply:
> HIGHPRIO is by definition the highest priority supported by the ChibiOS kernel IIRC
 
I didn't know that, seems OK.

> We need the highest priority here to avoid dropping messagebus messages, although that might be acceptable.

Not acceptable to drop messages? :thinking: 

> I don't think the odometry thread will be starved by this more than anything, the position manager is already running at NORMALPRIO, along with everything else.

Fair enough, can we verify this when running a hardware test?

> Sure, but then we should file a bug and someone should implement it.

Ask no more: #265